components/availability/src/Availability.svelte

```diff
@@ -160,27 +195,29 @@
 
 <nylas-error {id} />
 <main bind:this={main}>
-  {#each days as day}
-    <div class="day">
-      <header>
-        <h2>{new Date(day.timestamp).toLocaleDateString()}</h2>
-      </header>
-      <ul class="slots">
-        {#each day.slots as slot}
-          <li
-            class="slot {slot.status}"
-            on:click={() => {
-              slot.status = handleTimeSlotClick(slot);
-            }}
-            on:mouseover={() =>
-              console.log(new Date(slot.time).toLocaleTimeString())}
-          >
-            <span>{new Date(slot.time).toLocaleString()}</span>
-          </li>
-        {/each}
-      </ul>
-    </div>
-  {/each}
+  <div class="days">
+    {#each days as day}
+      <div class="day">
+        <header>
+          <h2>{new Date(day.timestamp).toLocaleDateString()}</h2>
+        </header>
+        <ul class="slots">
+          {#each day.slots as slot}
+            <li
+              class="slot {slot.selectionStatus} {slot.availability}"
+              on:click={() => {
+                slot.selectionStatus = handleTimeSlotClick(slot);
+              }}
```

Comment:
This looks like it's only accessible to mouse users as there's no tabindex attribute. Trying adding that and testing that it can be tabbed to?

Reply:
Thanks for this -- addressing now.`<li>` to `<button>` incoming.